soccer/src/soccer/radio/soccer_mom.cpp

```diff
@@ -0,0 +1,27 @@
+// use tutorial name space /referee/team_color publish to /team_fruit
+
+#include "soccer_mom.hpp"
+#include <spdlog/spdlog.h>
+
+
+namespace tutorial {
+    SoccerMom::SoccerMom() : rclcpp::Node{"soccer_mom"} {
+                team_fruit_pub_ = create_publisher<std_msgs::msg::String>(
+                    "team_fruit", rclcpp::QoS(1));
+
+                team_color_sub_ = create_subscription<rj_msgs::msg::TeamColor>(
+                    referee::topics::kTeamColorTopic, rclcpp::QoS(1).transient_local(),
```

Comment:
`transient_local()` is not harmful, but take a minute to read up on what it means. in general, you don't usually need it